Mcc Student Bistro Moqups

```diff
@@ -0,0 +1,2 @@
+https://app.moqups.com/P8biiagsUo/edit/page/ad64222d5
+This is my Moqups Thing!
```

Comment:
Awesome! Thank you for including this!

Reply:
It says I don't have access - so you would want to make sure that you have the share settings set to public going forward.